clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/EventualServerStoreProxy.java

```diff
@@ -111,11 +160,14 @@ public Chain get(long key) throws TimeoutException {
   @Override
   public void append(final long key, final ByteBuffer payLoad) throws TimeoutException {
     delegate.append(key, payLoad);
+    incrementInvalidationsCounter(key);
   }
 
   @Override
   public Chain getAndAppend(final long key, final ByteBuffer payLoad) throws TimeoutException {
-    return delegate.getAndAppend(key, payLoad);
+    Chain chain = delegate.getAndAppend(key, payLoad);
```

Comment:
Nitpicking here: Did you intend to do something with that returned chain? Otherwise I don't see why it's kept? Otherwise the change set could have been just:
```java
incrementInvalidationsCounter(key);
return delegate.getAndAppend(key, payLoad);
```
right?